storybookjs/storybook
# Select knob key/value ordering

continued from https://github.com/storybooks/storybook/pull/1449

fixes #799

Issue:

Using knobs is currently confusing. 

Example, for complex strings and null values

```
select('My Select', {
  none: null, // renders empty item
  [`Computed key`]: 'computed', 
  'Some kind of long sentence. This is ok, mainly just looks odd as a key': 'sentence',
})
```


## What I did

Use value as value
Still uses key for the key property on each `<option>`

## How to test

```
select('My Select', {
  Empty: null,
  'First item': 'The first item selected',
  second: `The item with key 'second' is selected`
})
```



Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/storybooks/storybook/1745)
